docs/api-reference/next/image.md

```diff
@@ -288,13 +288,16 @@ module.exports = {
 }
 ```
 
+### decoding
+
+Defaults to `"async"` but can be overridden [to `"auto"` or `"sync"`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/decoding) if necessary (though this may impact performance).
```

Comment:
Can we provide more guidance when to change this?

Also would you want to mix `loading="lazy"` or `decoding="sync"` or should we error in that case?

Reply:
Another question: should setting `loader="eager"` change the default value of `decoding`?

Reply:
@styfle theres no real reason not to allow mixing these. In our case we lazy load images below the fold. All images have a pregenerated blur placeholder and could therefore also decode async. If thinking about a slider, I would rather decode sync, as the „next“ slide should probably be decoded asap, while still initiating loading only when the slide becomes visible.

Reply:
@styfle as Next image is already massively complex in its nature and default settings I would not add more complexity by introducing side effects like „when this then that“. Allow developers to handle advanced props like these without side chaining decoding to loading logic